lib/src/bloc/bloc.dart

```diff
@@ -0,0 +1,4 @@
+abstract class Bloc {
+  void initialize();
+  void dispose();
+}
```

Comment:
missed an extra line here, do Ctrl + Alt + L to fix it or do it manually

Reply:
fixed